src/Generator/DomDocument.php

```diff
@@ -31,11 +31,12 @@ protected function openXmlElement($name)
     /**
      * Write XML text data into the currently opened XML element
      *
-     * @param string $text
+     * @param mixed $text
```

Comment:
I'd argue the annotation should remain the same here; we're still expecting a string, but now you're explicitly casting it in the method to ensure we don't get errors. (A later PR can then add the explicit parameter type.)